.github/workflows/ci_trie_benchmark.yml

```diff
@@ -0,0 +1,42 @@
+name: trie benchmark
+
+on:
+  workflow_call:
+  pull_request:
+    branches: ["**"]
+    paths:
+      - "crates/l2/**"
+
+permissions:
+  # deployments permission to deploy GitHub pages website
+  deployments: write
+  # contents permission to update benchmark contents in gh-pages branch
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@1.85.0
+      - uses: actions/checkout@v4
+        with:
+          lfs: true
+      - name: Checkout LFS objects
+        run: git lfs checkout
+      - name: Build benchmarks
+        run: |
+          cd crates/common/trie
+          make build-ethrex-bench
+          make build-cita-bench
+      - name: Run benchmarks and save output
+        run: |
+          make compare
```

Comment:
Comparing cita tree against ethrex is useful, but what's really useful is comparing ethrex base vs ethrex pr (how the pr changes affect the performance). Maybe the ideal would be

Ethrex Pr
Ethrex Base
Cita (base or pr or cache, doesn't matter).

Reply:
Sure, let's do that